### PR TITLE
Translate docs/helm/helm_get in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>

Translate docs/helm/helm_get in Korean
- https://helm.sh/docs/helm/helm_get/

### CODE REVIEW

**Comment by bluefriday** [comment]
@jmnote @jmyung 리뷰 요청 드립니다.

File: content/ko/docs/helm/helm_get_hooks.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### 부모 명령어에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  표준 오류를 로그 및 파일로 표시
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:36]:
```suggestion
      --kube-token string                인증에 사용할 베어러(bearer) 토큰
```

File: content/ko/docs/helm/helm_get_notes.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+### 부모 명령어에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  표준 오류를 로그 및 파일로 표시
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:34]:
```suggestion
      --kube-token string                인증에 사용할 베어러(bearer) 토큰
```

File: content/ko/docs/helm/helm_get_values.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+helm get values RELEASE_NAME [flags]
+```
+
+### 옵션
+
+```
+  -a, --all             모든 (계산 된) 값 덤프
+  -h, --help            helm get values 에 대한 도움말
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:23]:
```suggestion
  -h, --help            values 에 대한 도움말
```

File: content/ko/docs/helm/helm_get_notes.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:44]:
```suggestion
      --repository-config string         저장소 이름 및 URL을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
```

File: content/ko/docs/helm/helm_get_values.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:46]:
```suggestion
      --repository-config string         저장소 이름 및 URL을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
```

File: content/ko/docs/helm/helm_get.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
+      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
+      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get.md:48]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_all.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
+      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
+      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_all.md:49]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_hooks.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
+      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
+      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:49]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_manifest.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
+      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
+      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:51]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_notes.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
+      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:47]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_values.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
+      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:49]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+  -v, --v Level                          로그 수준 상세표시 레벨
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get.md:49]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_all.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+  -v, --v Level                          로그 수준 상세표시 레벨
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_all.md:50]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_hooks.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+  -v, --v Level                          로그 수준 상세표시 레벨
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:50]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_manifest.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+  -v, --v Level                          로그 수준 상세표시 레벨
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:52]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_notes.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+  -v, --v Level                          로그 수준 상세표시 레벨
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:48]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_values.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+  -v, --v Level                          로그 수준 상세표시 레벨
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:50]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_hooks.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+## helm get hooks
+
+명명된 릴리스에 대한 모든 훅 다운로드
+
+### 개요
+
+
+이 명령은 지정된 릴리스에 대한 훅을 다운로드한다.
+
+훅은 YAML로 형식이 지정되고 YAML '---\n' 구분자로 구분된다.
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:14]:
```suggestion
훅은 YAML로 형식이며 YAML '---\n' 구분자로 구분된다.
```

File: content/ko/docs/helm/helm_get_hooks.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+## helm get hooks
+
+명명된 릴리스에 대한 모든 훅 다운로드
+
+### 개요
+
+
+이 명령은 지정된 릴리스에 대한 훅을 다운로드한다.
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:12]:
```suggestion
이 명령어는 지정한 릴리스에 대한 훅을 다운로드한다.
```

File: content/ko/docs/helm/helm_get_manifest.md [hunk 20]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+## helm get manifest
+
+명명된 릴리스에 대한 매니페스트 다운로드
+
+### 개요
+
+
+이 명령어는 지정된 릴리스에 대해 생성된 매니페스트를 가져온다.
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:12]:
```suggestion
이 명령어는 지정한 릴리스에 대해 생성된 매니페스트를 가져온다.
```

File: content/ko/docs/helm/helm_get_values.md [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+```
+helm get values RELEASE_NAME [flags]
+```
+
+### 옵션
+
+```
+  -a, --all             모든 (계산 된) 값 덤프
+  -h, --help            helm get values 에 대한 도움말
+  -o, --output format   지정된 형식으로 출력을 표시. 허용되는 값: table, json, yaml (기본값 table)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:24]:
```suggestion
  -o, --output format   지정한 형식으로 출력을 표시. 허용되는 값: table, json, yaml (기본값 table)
```

File: content/ko/docs/helm/helm_get_hooks.md [hunk 22]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+helm get hooks RELEASE_NAME [flags]
+```
+
+### 옵션
+
+```
+  -h, --help           helm get hooks 에 대한 도움말
+      --revision int   개정판으로 명명된 릴리스 가져오기
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:25]:
```suggestion
      --revision int   리비전으로 명명된 릴리스 가져오기
```

File: content/ko/docs/helm/helm_get_manifest.md [hunk 23]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+helm get manifest RELEASE_NAME [flags]
+```
+
+### 옵션
+
+```
+  -h, --help           helm get manifest 에 대한 도움말
+      --revision int   개정판으로 명명된 릴리스 가져오기
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:27]:
```suggestion
      --revision int   리비전으로 명명된 릴리스 가져오기
```

File: content/ko/docs/helm/helm_get_notes.md [hunk 24]
```diff
@@ -0,0 +1,56 @@
...
+
+```
+helm get notes RELEASE_NAME [flags]
+```
+
+### 옵션
+
+```
+  -h, --help           helm get notes 에 대한 도움말
+      --revision int   개정판으로 명명된 릴리스 가져오기
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:23]:
```suggestion
      --revision int   리비전으로 명명된 릴리스 가져오기
```

File: content/ko/docs/helm/helm_get_values.md [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+      --revision int    개정판으로 명명된 릴리스 가져오기
```
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:25]:
```suggestion
      --revision int    리비전으로 명명된 릴리스 가져오기
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다.

**Comment by bluefriday** [comment]
> 수고하셨습니다.

@jmnote 리뷰 감사드립니다.

**Comment by jmnote** [comment]
lgtm

File: content/ko/docs/helm/helm_get.md [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+      --kube-token string                인증에 사용할 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get.md:44]: Are you sure about this file path change? Or is this an accident?

https://github.com/helm/helm-www/blame/master/content/en/docs/helm/helm_get.md#L44 says:

 --repository-cache string          path to the file containing cached repository indexes (default "~/.cache/helm/repository")

File: content/ko/docs/helm/helm_get_all.md [hunk 27]
```diff
@@ -0,0 +1,58 @@
...
+      --kube-token string                인증에 사용할 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_all.md:45]: Are you sure about this file path change? Or is this an accident?

https://github.com/helm/helm-www/blame/master/content/en/docs/helm/helm_get_all.md#L45 says:

 --repository-cache string          path to the file containing cached repository indexes (default "~/.cache/helm/repository")

File: content/ko/docs/helm/helm_get_hooks.md [hunk 28]
```diff
@@ -0,0 +1,58 @@
...
+      --kube-token string                인증에 사용할 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_hooks.md:45]: Please check this default path (see other comments).

File: content/ko/docs/helm/helm_get_manifest.md [hunk 29]
```diff
@@ -0,0 +1,60 @@
...
+      --kube-token string                인증에 사용할 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_manifest.md:47]: Please check this default path (see other comments).

File: content/ko/docs/helm/helm_get_notes.md [hunk 30]
```diff
@@ -0,0 +1,56 @@
...
+      --kube-token string                인증에 사용할 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_notes.md:43]: Please check this default path (see other comments).

File: content/ko/docs/helm/helm_get_values.md [hunk 31]
```diff
@@ -0,0 +1,58 @@
...
+      --kube-token string                인증에 사용할 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_values.md:45]: Please check this default path (see other comments).

**Comment by bridgetkromhout** [comment]
This LGTM once we fix the file path (which I think changed accidentally?)

File: content/ko/docs/helm/helm_get.md (hunk 26 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_get.md:44]: It's my mistake. 
In previous I translated 'helm' command  (https://github.com/helm/helm-www/pull/802).
And in that docs, 'repository cache string path' was set to '~/snap/code/common/.cache/helm/repository'.
After the PR was merged, I used the same sentence for uniformity in translation.

But I couldn't confirm that only that path is different from other commands.
I'll fix that in every commit I've PRed. Thank you.

File: content/ko/docs/helm/helm_get_all.md (hunk 27 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_all.md:45]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_hooks.md (hunk 28 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_hooks.md:45]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_manifest.md (hunk 29 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_manifest.md:47]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_notes.md (hunk 30 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_notes.md:43]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_values.md (hunk 31 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_values.md:45]: I will modify the same as above.

**Comment by bluefriday** [comment]
> This LGTM once we fix the file path (which I think changed accidentally?)

@bridgetkromhout  I modified it according to contents you reviewed.

**Review by bridgetkromhout — APPROVED** [review]
LGTM

**Pull request merged**